.tributors

```diff
@@ -2573,11 +3064,24 @@
         "name": "Rémi Adon"
     },
     "samirdas": {
+        "affiliation": "Indian Council of Agricultural Research",
         "blog": "https://github.com/samirdas",
         "contributions": [
             "doc"
         ],
-        "name": "Samir Das"
+        "name": "Samir Das",
+        "orcid": "0000-0002-6613-7347"
+    },
+    "sdiebolt": {
+        "blog": "https://samuel.diebolt.io/",
+        "contributions": [
+            "doc",
+            "ideas"
+        ],
+        "email": "samuel.diebolt@iconeus.com",
+        "github": "sdiebolt",
+        "name": "Samuel Diebolt",
+        "orcid": "0000-0002-9788-7263"
```

Comment:
```suggestion
    "sdiebolt": {
        "blog": "https://samuel.diebolt.io/",
        "contributions": [
            "doc",
            "ideas"
        ],
        "email": "samuel@diebolt.io",
        "github": "sdiebolt",
        "name": "Samuel Le Meur-Diebolt",
        "orcid": "0000-0002-9788-7263"
```

Reply:
Moved to personal email while waiting for my new affiliation's email, and added new last name. Apart for that, everything's alright!